editor/project.clj

```diff
@@ -225,6 +225,7 @@
                       :no-schemas {:jvm-opts ["-Ddefold.schema.check.disable=true"]}
                       :no-spec-asserts {:jvm-opts ["-Dclojure.spec.compile-asserts=false"]}
                       :performance [:no-decorated-exceptions :no-schemas :no-spec-asserts]
+                      :16gb {:jvm-opts ["-Xmx16g"]}
```

Comment:
usually when we are talking about 16GB, it's about the laptop memory, and as I remember we allow jvm to use only 75% out of it, so it should be something like 12gb

Reply:
Good point, I'll try to use even lower heaps.